### PR TITLE
promote-config: promote next-devel to next

### PR DESCRIPTION
**Pull request by jlebon**
Fedora 35 Beta is now GO. Start shipping from `next-devel`.

### CODE REVIEW

**Comment by jlebon** [comment]
Hold this until Fedora 35 Beta is actually GO.

Next GO/NO-GO meeting is next week:
https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/EMLQZF6CMPQUPE6AQ4QWCVDB5SOXHFWX/

We should decide whether to hold `next` next week until Thursday or just catch it on the next rounds.

**Comment by dustymabe** [comment]
> We should decide whether to hold `next` next week until Thursday or just catch it on the next rounds.

Even if we held it until Thursday beta wouldn't ship til the following Tuesday. I say we just wait until it's actually shipped.

**Comment by jlebon** [comment]
> Even if we held it until Thursday beta wouldn't ship til the following Tuesday.

:+1: 

> I say we just wait until it's actually shipped.

I think it'd be great if we shipped on the day or the day after Beta was out. Though the question is whether we still do the regular release next week. Leaning towards yes, because we don't actually know when Beta will be out.

**Comment by dustymabe** [comment]
> I think it'd be great if we shipped on the day or the day after Beta was out.

Yeah. I think that would be ideal. No reason we can't ship on the day because we'll know on the Thursday before and the content set shouldn't change.

> Though the question is whether we still do the regular release next week. Leaning towards yes, because we don't actually know when Beta will be out.

Agree. I don't mind doing multiple releases in `next`. They are about to start being weekly anyway as we get closer to F35.

**Review by dustymabe — APPROVED** [review]

**Pull request merged**
